extension.js

```diff
@@ -35,6 +35,8 @@ require('fs').writeFileSync('/Users/musa/easy-extensibility/E-snippets.json', JS
 // vscode, E, commands ================================================================================
 
 const vscode = require('vscode')
+const conf = vscode.workspace.getConfiguration("easy-extensibility").get
+const rcfile = conf("rcFile")
```

Comment:
It looks like you're using `rcFile` as the config setting to avoid the hardcoded name `init.js`.

1. Why the name `rcFile`? In particular, what does  the prefix`rc` stand for?
2. Would you please tell me a bit more about line 38. Does this get our/easy-extensibility's internal configuration or the users `settings.json`?

Reply:
Please refer to [https://stackoverflow.com/questions/11030552/what-does-rc-mean-in-dot-files](url)
The ‘rc’ suffix goes back to Unix's grandparent, CTSS. It had a command-script feature called "runcom". Early Unixes used ‘rc’ for the name of the operating system's boot script, as a tribute to CTSS runcom.